builtin/k8s/deployment.go

```diff
@@ -45,4 +45,8 @@ func (d *Deployment) newDeployment(name string) *appsv1.Deployment {
 	}
 }
 
+func (d *Deployment) URL() string {
+	return d.Url
```

Comment:
Maybe I'm being dense, but where are all these being populated?

Reply:
It's not being populated in fact. This is just an empty URL at the moment, waiting to be implemented:
https://github.com/hashicorp/waypoint/pull/1387/files/1dcacbb891bf4f181adfd3c952a96373073648be#diff-1b0cb4abccfe10f5d0dcb1e343487a8da3a3da4c7b06dd4e86be69b5456681f1R10